NoSuchMethodError: No static method toHumanReadableAscii(Ljava/lang/String;)Ljava/lang/String; in class Lokhttp3/internal/Util

Crash when opening Nearby (on browse branch, Android 7.1.2)
```
06-23 20:45:33.741 11594 11594 E AndroidRuntime: FATAL EXCEPTION: main
06-23 20:45:33.741 11594 11594 E AndroidRuntime: Process: fr.free.nrw.commons.debug, PID: 11594
06-23 20:45:33.741 11594 11594 E AndroidRuntime: java.lang.NoSuchMethodError: No static method toHumanReadableAscii(Ljava/lang/String;)Ljava/lang/String; in class Lokhttp3/internal/Util; or its super classes (declaration of 'okhttp3.internal.Util' appears in base.apk)
06-23 20:45:33.741 11594 11594 E AndroidRuntime:        at com.mapbox.services.android.telemetry.MapboxTelemetry.setUserAgent(MapboxTelemetry.java:315)
06-23 20:45:33.741 11594 11594 E AndroidRuntime:        at com.mapbox.services.android.telemetry.MapboxTelemetry.initialize(MapboxTelemetry.java:174)
06-23 20:45:33.741 11594 11594 E AndroidRuntime:        at com.mapbox.services.android.telemetry.MapboxTelemetry.initialize(MapboxTelemetry.java:125)
06-23 20:45:33.741 11594 11594 E AndroidRuntime:        at com.mapbox.mapboxsdk.Mapbox.getInstance(Mapbox.java:60)
06-23 20:45:33.741 11594 11594 E AndroidRuntime:        at fr.free.nrw.commons.nearby.NearbyMapFragment.onCreate(NearbyMapFragment.java:162)
06-23 20:45:33.741 11594 11594 E AndroidRuntime:        at android.support.v4.app.Fragment.performCreate(Fragment.java:2331)
06-23 20:45:33.741 11594 11594 E AndroidRuntime:        at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1386)
06-23 20:45:33.741 11594 11594 E AndroidRuntime:        at android.support.v4.app.FragmentTransition.addToFirstInLastOut(FragmentTransition.java:1188)
06-23 20:45:33.741 11594 11594 E AndroidRuntime:        at android.support.v4.app.FragmentTransition.calculateFragments(FragmentTransition.java:1071)
06-23 20:45:33.741 11594 11594 E AndroidRuntime:        at android.support.v4.app.FragmentTransition.startTransitions(FragmentTransition.java:115)
06-23 20:45:33.741 11594 11594 E AndroidRuntime:        at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2380)
06-23 20:45:33.741 11594 11594 E AndroidRuntime:        at android.support.v4.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2338)
06-23 20:45:33.741 11594 11594 E AndroidRuntime:        at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2245)
06-23 20:45:33.741 11594 11594 E AndroidRuntime:        at android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:703)
06-23 20:45:33.741 11594 11594 E AndroidRuntime:        at android.os.Handler.handleCallback(Handler.java:751)
06-23 20:45:33.741 11594 11594 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:95)
06-23 20:45:33.741 11594 11594 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:154)
06-23 20:45:33.741 11594 11594 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:6186)
06-23 20:45:33.741 11594 11594 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
06-23 20:45:33.741 11594 11594 E AndroidRuntime:        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
06-23 20:45:33.741 11594 11594 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
06-23 20:45:33.750   786  6707 W ActivityManager:   Force finishing activity fr.free.nrw.commons.debug/fr.free.nrw.commons.nearby.NearbyActivity
```

I tried upgrading MapBox to 6.2.0 but get this error:
```
/home/nico/src/commons-upstream/app/src/main/java/fr/free/nrw/commons/nearby/NearbyMapFragment.java:46: error: package com.mapbox.services.android.telemetry does not exist
import com.mapbox.services.android.telemetry.MapboxTelemetry;
```

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -22,7 +22,7 @@ dependencies {
     implementation 'info.debatty:java-string-similarity:0.24'
     implementation 'com.borjabravo:readmoretextview:2.1.0'
     implementation 'com.android.support.constraint:constraint-layout:1.0.2'
-    implementation('com.mapbox.mapboxsdk:mapbox-android-sdk:5.4.1@aar') {
+    implementation('com.mapbox.mapboxsdk:mapbox-android-sdk:5.5.0@aar') {
         transitive = true
     }
     implementation 'com.github.deano2390:MaterialShowcaseView:1.2.0'

